### PR TITLE
adds css framework to html

### PR DESCRIPTION
**Pull request by amlorpants**

### CODE REVIEW

File: assets/script.js
Comment (amlorpants) [inline assets/script.js:1]: This is the api tester boilerplate that I left in slack

File: index.html
Comment (amlorpants) [inline index.html:1]: this is all of the BASIC css framework for our original design. NO custom css yet just items to call to in javascript.

File: assets/images/viva-luna-studios-r_-onuwuWAU-unsplash.jpg
Comment (amlorpants) [inline assets/images/viva-luna-studios-r_-onuwuWAU-unsplash.jpg:1]: this is a image placeholder for our main page

**Review by RossiniTheRat — APPROVED** [review]
Looks good

**Pull request merged**